### PR TITLE
Nerfs sniper vision

### PR DESCRIPTION
**Pull request by Pratepresidenten**
Removes the ability to snipe-examine items from a mile away.
It now requires you to be within grabbing distance of the item to access that sweet, sweet description.

### CODE REVIEW

**Comment by Pratepresidenten** [comment]
!review

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -140,24 +140,28 @@
...
+			if (ITEMSIZE_NORMAL to ITEMSIZE_LARGE)
+				size = "normal-sized"
+			if (ITEMSIZE_SMALL to ITEMSIZE_NORMAL)
+				size = "small"
+			if (0 to ITEMSIZE_TINY)
+				size = "tiny"
+
+		return ..(user, distance, "", "It is \a [size] item.")
+	else 
+		to_chat(user, "Thats \a [name].")
```
Comment (NonQueueingMatt) [inline code/game/objects/items.dm]:
```suggestion
		to_chat(user, "\icon[src] That's \a [name].")
```
This proc won't play nice with blood stained or oil stained items.

**Comment by Pratepresidenten** [comment]
I will get on this soon:tm:

**Comment by Pratepresidenten** [comment]
!wip

**Comment by NonQueueingMatt** [comment]
Author inactivity and merge conflicts, reopen when ready.

**Pull request closed without merging**